Makefile

```diff
@@ -56,3 +56,6 @@ serve-dev-draft: setup
 
 serve-dev-future: setup
 	hugo server -F $(DEV_OPTIONS) $(SERVE_OPTIONS)
+
+prettierfix: setup
```

Comment:
I added this as I have not succeeded in configuring VSCode's prettier plugin & this seems like a much more reproducible way to get prettier to fix docs for folks that are only contributing to docs occasionally.

Reply:
Thank you!